cohort-engine/src/main/java/com/ibm/cohort/engine/cqfruler/MeasureSupplementalDataEvaluation.java

```diff
@@ -63,15 +65,19 @@ public static void populateSDEAccumulators(Context context, Patient patient,
 						code = ((Code) sdeListItem).getCode();
 						break;
 					case "ArrayList":
-						if (!((ArrayList<?>) sdeListItem).isEmpty()) {
+						if (((ArrayList<?>) sdeListItem).isEmpty()) {
+							return;
```

Comment:
I'm not certain  about the expected behavior of this code, but does this update change the behavior? Will `return`  skip over the rest of the items in `sdeList`, whereas `continue` would have just gone to the next item?

Reply:
Good catch, realistically I'm not convinced we should hit that case anyway (thanks cqf-ruler for 0 tests and examples) but I rejiggered the code.